test/sdk/ThreadTest.ooc

```diff
@@ -0,0 +1,77 @@
+use ooc-unit
+use ooc-base
+import threading/Thread
+import threading/native/ConditionUnix
+import os/Time
+
+IntObject: class {
```

Comment:
Closures didn't work with integers, it seems like a separate local copy was created instead of modifying the original integer, so I created this helper class.